SPECS/azurelinux-rpm-macros/azurelinux-rpm-macros.spec

```diff
@@ -138,6 +143,9 @@ install -p -m 644 -t %{buildroot}%{rcluadir}/srpm python.lua
 %{_rpmconfigdir}/macros.d/macros.check
 
 %changelog
+* Tue Aug 13 2024 Daniel McIlvaney <damcilva@microsoft.com> - %{azl}.0-6
```

Comment:
does the %{azl} macro work in the changelog? might be simpler to have `3.0-6`

Reply:
```suggestion
* Tue Aug 13 2024 Daniel McIlvaney <damcilva@microsoft.com> - 3.0-6
```

Reply:
I used `rpmdev-bumpspec` to manage the .spec, and yes afaik it does work, but I'll change it back.